lemur/migrations/versions/2201c548a5a1_.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
```

Comment:
Could you add a description here? I know the migration is fairly self-descriptive, but I think anything is better than `empty message` :)

Reply:
excellent point. definitely plus one!